google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Query.java

```diff
@@ -59,8 +59,19 @@ public static Query create(String tableId) {
     return new Query(tableId);
```

Comment:
Same here, we can convert tableId to a TargetId here and call the TargetId ctor, and remove the existing ctor.